src/vanilla/store.ts

```diff
@@ -772,6 +802,8 @@ export const createStore = () => {
           l({ type: 'restore', flushed: flushed as Set<AnyAtom> }),
         )
       },
+      dev_print_call_counts: printCounts,
+      dev_clear_call_counts: resetCounts,
     }
```

Comment:
is this necessary for our core library? I think making a wrapper in user side should make more sense